src/tigerbeetle/main.zig

```diff
@@ -349,6 +349,16 @@ const Command = struct {
 
         const clients_limit = constants.pipeline_prepare_queue_max + args.pipeline_requests_limit;
 
+        const trace_file = if (args.trace) |trace_path|
+            std.fs.cwd().createFile(trace_path, .{ .exclusive = true }) catch |err| {
+                std.debug.panic("error creating trace file: {}", .{err});
+            }
+        else
+            null;
+        errdefer if (trace_file) |file| file.close();
```

Comment:
Should we always close this file at the end? (although we don't have a graceful shutdown to exit `main` :thinking:)
```suggestion
        defer if (trace_file) |file| file.close();
```